kolibri/plugins/coach/assets/src/views/common/resourceSelection/UpdatedResourceSelection.vue

```diff
@@ -21,7 +21,7 @@
         :contentCheckboxDisabled="contentCheckboxDisabled"
         :contentCardLink="contentLink"
         :contentCardMessage="contentCardMessage"
-        :showRadioButtons="!multi"
+        :showRadioButtons="showRadioButton"
```

Comment:
I'm a little bit confused about what's happening here with the radio buttons. My thinking is that maybe there is some pre-existing 0.17 work that allows for selection for practice quizzes, that is colliding a bit with this new scenario, of making individual questions selectable. I realized in the bug bash last week that I hadn't fully specced out the updates to practice quizzes, and I think that can be done in follow up since it's about selection of resources in general, not just in search results. But here in your work, we do want to have the selection be a checkbox (if selecting the whole resource) rather than a radio button

Reply:
If there's anything you want to chat through about this, let me know and we can outline together what the scenarios need to be :)

Reply:
It's kinder confusing because I see on the spec it's a radio button.  Since the goal here is to allow selecting individual questions, switching to checkboxes for whole-resource selection sounds like the right approach.

Reply:
I think understanding the context of the `multi` prop would be a great starting point. From a quick read through, the `showRadioButton` is some kind of a toggle between the use of radio or checkboxes. I guess its mean to offer flexibility for the card. 

The `multi` props context in the `SelectFromBookmarks.vue` and `SelectFromTopics.vue` is `!settings?.selectPracticeQuiz`(which i think means practice quiz is selectable or not?) so using `target === SelectionTarget.QUIZ` could break functionality, I think? 

But back to Marcella's point, its probably best to chat about this "in-person" as it could involve a few moving pieces. Hopefully that helps